app/controllers/concerns/idv/phone_otp_rate_limitable.rb

```diff
@@ -35,6 +35,7 @@ def handle_too_many_otp_sends
 
     def handle_too_many_otp_attempts
       analytics.idv_phone_confirmation_otp_rate_limit_attempts
+      irs_attempts_api_tracker.idv_phone_otp_submitted_rate_limited
```

Comment:
Adding this here because the `analytics_tracker` is logging a similar event here.